src/main/kotlin/minesweeper/Location.kt

```diff
@@ -0,0 +1,20 @@
+package minesweeper
+
+class Location private constructor(private val y: Int, private val x: Int) {
+
+    fun getConvertIndex(width: Int): Int {
+        return y * width + x
+    }
+
+    companion object {
+        fun of(location: String?): Location {
+            require(location != null && location.isNotBlank()) { "빈 값을 입력하면 안됩니다." }
+            val locationPart = location.filter { !it.isWhitespace() }.split(",")
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
            val locationPart = location.split(",").map { it.trim() }
```

Reply:
감사합니다! 반영했습니다!